setup.py

```diff
@@ -24,7 +24,12 @@
 # Requirements should be as minimal as possible.
 # Avoid pinning, and use minimum version numbers
 # only where required.
-requirements = ["pennylane>=0.23", "requests", "pyjwt", "toml"]
+requirements = [
+    "pennylane",

```

Comment:
iirc there is a way to pin to master in setup.py as well

Reply:
Yes you can check my Qiskit PR if needed